bin/enqueue.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+if [ ! -f DeepSpeech.py ]; then
+  echo "Please make sure you run this from DeepSpeech's top level directory."
+  exit 1
+fi
+
+GLOBAL_LOG="\\/dev\\/null" # by default there is no global logging
+NODES=2 # to be adjusted
+GPUS=8 # to be adjusted
```

Comment:
Right, I don't mind that they're defaults, so much as to change them you need to modify this script. Can't they be command-line parameters?